### ISSUE TITLE
After upgrade to QTS > 5.0.1.2145 Tailscale is not compatible anymore

### ISSUE THREAD

**Issue by scheini1972** [tailscale/tailscale-qpkg#46]
I just upgraded to the latest firmware from Qnap, 5.0.1.2145.
Unfortunately, tailscale (latest version 1.30.2) is not compatible anymore with it.
:-(

**Comment by case** [comment]
@scheini1972 can you share the errors or logs output you're seeing?

**Comment by ivokub** [comment]
Can you elaborate? Doesn't start anymore?

What is the output of
```
./tailscaled --port 41641 --state=var/lib/tailscale/tailscaled.state --socket=var/run/tailscale/tailscaled.sock
```

**Comment by wingcomm** [comment]
Yes, I have the same issue. This package no longer works properly for some reason. Running that command does bring Tailscale up, but the daemon won't run/start since the upgrade.

```
[/share/CACHEDEV1_DATA/.qpkg/Tailscale] # ./tailscale -socket var/run/tailscale/tailscaled.sock status
failed to connect to local tailscaled; it doesn't appear to be running (sudo systemctl start tailscaled ?)
```

Here is the output that you requested @ivokub (I did hit Ctrl-C towards the end):
 
```
[/share/CACHEDEV1_DATA/.qpkg/Tailscale] # ./tailscaled --port 41641 --state=var/lib/tailscale/tailscaled.state --socket=var/run/tailscale/tailscaled.sock
logtail started
Program starting: v1.30.1-t949c40030, Go 1.19.1-tsb13188d: []string{"./tailscaled", "--port", "41641", "--state=var/lib/tailscale/tailscaled.state", "--socket=var/run/tailscale/tailscaled.sock"}
LogID: 8d063514f7fdb302ef8687390efad2e5f67a090a687e7f2b167235053a70ccc7
logpolicy: using system state directory "/var/lib/tailscale"
wgengine.NewUserspaceEngine(tun "tailscale0") ...
router: disabling tunneled IPv6 due to system IPv6 config: disable_ipv6 is set
dns: [rc=unknown ret=direct]
dns: using *dns.directManager
link state: interfaces.State{defaultRoute=br0 ifs={br0:[192.168.1.104/24] docker0:[10.0.5.1/24] lxcbr0:[10.0.3.1/24] lxdbr0:[10.0.7.1/24]} v4=true v6=false}
magicsock: disco key = d:d9838cf9f4a5a676
Creating WireGuard device...
Bringing WireGuard device up...
Bringing router up...
external route: up
Clearing router settings...
Starting link monitor...
Engine created.
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:254 Protocol:0 Scope:0 Type:1 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5210 Table:254 Mark:4294967295 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:253 Protocol:0 Scope:0 Type:1 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5230 Table:253 Mark:4294967295 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:0 Protocol:0 Scope:0 Type:7 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5250 Table:0 Mark:4294967295 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:52 Protocol:0 Scope:0 Type:1 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5270 Table:52 Mark:0 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
network-lock unavailable; no state directory
qnap Taildrop support: shared folder "Taildrop" not found
Start
using backend prefs for "_daemon": Prefs{ra=false dns=true want=true routes=[] nf=on Persist{lm=, o=[5Nu2H], n=[pq7KP] u="rweir@viconcepts.com"}}
Backend: logs: be:8d063514f7fdb302ef8687390efad2e5f67a090a687e7f2b167235053a70ccc7 fe:
control: client.Login(false, 0)
control: doLogin(regen=false, hasUrl=false)
health("overall"): error: not in map poll
router: somebody (likely systemd-networkd) deleted ip rules; restoring Tailscale's
trying bootstrapDNS("derp1e.tailscale.com", "64.225.56.166") for "controlplane.tailscale.com" ...
bootstrapDNS("derp1e.tailscale.com", "64.225.56.166") for "controlplane.tailscale.com" = [2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:201:7200:6340:b22f:7df6 18.156.90.224 18.193.143.177 3.121.104.141 18.193.255.254 52.29.8.43 18.157.173.201]
control: control server key from https://controlplane.tailscale.com: ts2021=[fSeS+], legacy=[nlFWp]
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
logtail: dial "log.tailscale.io:443" failed: dial tcp: lookup log.tailscale.io: no such host (in 20.029s), trying bootstrap...
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
trying bootstrapDNS("derp8d.tailscale.com", "178.62.44.132") for "controlplane.tailscale.com" ...
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
bootstrapDNS("derp8d.tailscale.com", "178.62.44.132") for "controlplane.tailscale.com" = [2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 18.193.255.254 18.157.173.201 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141]
trying bootstrapDNS("derp1d.tailscale.com", "165.22.33.71") for "controlplane.tailscale.com" ...
bootstrapDNS("derp1d.tailscale.com", "165.22.33.71") for "controlplane.tailscale.com" = [2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:24dd:46aa:f98e:9997 52.29.8.43 18.156.90.224 18.157.173.201 18.193.143.177 18.193.255.254 3.121.104.141]
trying bootstrapDNS("derp1c.tailscale.com", "104.248.8.210") for "log.tailscale.io" ...
bootstrapDNS("derp1c.tailscale.com", "104.248.8.210") for "log.tailscale.io" = [2600:1f18:429f:9305:4043:217b:512c:f8d4 34.229.201.48]
logtail: bootstrap dial succeeded
trying bootstrapDNS("derp9c.tailscale.com", "155.138.243.219") for "controlplane.tailscale.com" ...
trying bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" ...
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
bootstrapDNS("derp9c.tailscale.com", "155.138.243.219") for "controlplane.tailscale.com" = [2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 18.193.255.254 18.156.90.224 52.29.8.43 3.121.104.141 18.157.173.201 18.193.143.177]
bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" = [2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:201:7200:6340:b22f:7df6 18.193.255.254 18.157.173.201 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141]
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
trying bootstrapDNS("derp11.tailscale.com", "18.230.97.74") for "controlplane.tailscale.com" ...
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
bootstrapDNS("derp11.tailscale.com", "18.230.97.74") for "controlplane.tailscale.com" = [2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:24dd:46aa:f98e:9997 18.157.173.201 18.156.90.224 3.121.104.141 18.193.255.254 18.193.143.177 52.29.8.43]
trying bootstrapDNS("derp6.tailscale.com", "68.183.90.120") for "controlplane.tailscale.com" ...
bootstrapDNS("derp6.tailscale.com", "68.183.90.120") for "controlplane.tailscale.com" = [2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:24dd:46aa:f98e:9997 52.29.8.43 18.156.90.224 3.121.104.141 18.157.173.201 18.193.255.254 18.193.143.177]
trying bootstrapDNS("derp9b.tailscale.com", "144.202.67.195") for "controlplane.tailscale.com" ...
trying bootstrapDNS("derp11.tailscale.com", "18.230.97.74") for "controlplane.tailscale.com" ...
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
bootstrapDNS("derp9b.tailscale.com", "144.202.67.195") for "controlplane.tailscale.com" = [2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 18.156.90.224 18.193.143.177 52.29.8.43 18.193.255.254 18.157.173.201 3.121.104.141]
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
bootstrapDNS("derp11.tailscale.com", "18.230.97.74") for "controlplane.tailscale.com" = [2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:24dd:46aa:f98e:9997 18.157.173.201 18.156.90.224 3.121.104.141 18.193.255.254 18.193.143.177 52.29.8.43]
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
trying bootstrapDNS("derp2f.tailscale.com", "208.111.34.178") for "controlplane.tailscale.com" ...
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
bootstrapDNS("derp2f.tailscale.com", "208.111.34.178") for "controlplane.tailscale.com" = [2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 18.156.90.224 18.157.173.201 3.121.104.141 18.193.143.177 52.29.8.43 18.193.255.254]
trying bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" ...
bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" = [2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:201:7200:6340:b22f:7df6 18.193.255.254 18.157.173.201 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141]
trying bootstrapDNS("derp6.tailscale.com", "68.183.90.120") for "controlplane.tailscale.com" ...
trying bootstrapDNS("derp8d.tailscale.com", "178.62.44.132") for "controlplane.tailscale.com" ...
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
bootstrapDNS("derp8d.tailscale.com", "178.62.44.132") for "controlplane.tailscale.com" = [2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 18.193.255.254 18.157.173.201 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141]
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
bootstrapDNS("derp6.tailscale.com", "68.183.90.120") for "controlplane.tailscale.com" = [2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:24dd:46aa:f98e:9997 52.29.8.43 18.156.90.224 3.121.104.141 18.157.173.201 18.193.255.254 18.193.143.177]
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
trying bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" ...
bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" = [2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:201:7200:6340:b22f:7df6 18.193.255.254 18.157.173.201 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141]
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
trying bootstrapDNS("derp12b.tailscale.com", "45.63.71.144") for "controlplane.tailscale.com" ...
bootstrapDNS("derp12b.tailscale.com", "45.63.71.144") for "controlplane.tailscale.com" = [2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:201:7200:6340:b22f:7df6 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141 18.193.255.254 18.157.173.201]
trying bootstrapDNS("derp6.tailscale.com", "68.183.90.120") for "controlplane.tailscale.com" ...
trying bootstrapDNS("derp7.tailscale.com", "167.179.89.145") for "controlplane.tailscale.com" ...
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
bootstrapDNS("derp7.tailscale.com", "167.179.89.145") for "controlplane.tailscale.com" = [2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:6dd4:fc68:35ae:2b80 18.193.255.254 18.193.143.177 18.157.173.201 18.156.90.224 3.121.104.141 52.29.8.43]
bootstrapDNS("derp6.tailscale.com", "68.183.90.120") for "controlplane.tailscale.com" = [2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:24dd:46aa:f98e:9997 52.29.8.43 18.156.90.224 3.121.104.141 18.157.173.201 18.193.255.254 18.193.143.177]
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
Received error: register request: Post "https://controlplane.tailscale.com/machine/register": connection attempts aborted by context: context deadline exceeded
trying bootstrapDNS("derp2d.tailscale.com", "192.73.252.65") for "controlplane.tailscale.com" ...
bootstrapDNS("derp2d.tailscale.com", "192.73.252.65") for "controlplane.tailscale.com" = [2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 18.156.90.224 18.157.173.201 3.121.104.141 18.193.143.177 52.29.8.43 18.193.255.254]
trying bootstrapDNS("derp8c.tailscale.com", "206.189.16.32") for "controlplane.tailscale.com" ...
bootstrapDNS("derp8c.tailscale.com", "206.189.16.32") for "controlplane.tailscale.com" = [2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 18.193.255.254 18.157.173.201 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141]
control: doLogin(regen=false, hasUrl=false)
control: RegisterReq: onode=[5Nu2H] node=[pq7KP] fup=false
trying bootstrapDNS("derp3.tailscale.com", "68.183.179.66") for "controlplane.tailscale.com" ...
trying bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" ...
bootstrapDNS("derp8b.tailscale.com", "46.101.74.201") for "controlplane.tailscale.com" = [2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:201:7200:6340:b22f:7df6 18.193.255.254 18.157.173.201 18.156.90.224 52.29.8.43 18.193.143.177 3.121.104.141]
control: RegisterReq: got response; nodeKeyExpired=false, machineAuthorized=true; authURL=false
active login: nas01.viconcepts.com
Switching ipn state NoState -> Starting (WantRunning=true, nm=true)
magicsock: SetPrivateKey called (init)
wgengine: Reconfig: configuring userspace WireGuard config (with 0/12 peers)
wgengine: Reconfig: configuring router
wgengine: Reconfig: configuring DNS
dns: Set: {DefaultResolvers:[] Routes:{beta.tailscale.net.:[] ts.net.:[] viconcepts.com.beta.tailscale.net.:[]}+65arpa SearchDomains:[viconcepts.com.beta.tailscale.net.] Hosts:13}
dns: Resolvercfg: {Routes:{.:[127.0.1.1]} Hosts:13 LocalDomains:[viconcepts.com.beta.tailscale.net. beta.tailscale.net. ts.net.]+65arpa}
dns: OScfg: {Hosts:[] Nameservers:[100.100.100.100] SearchDomains:[viconcepts.com.beta.tailscale.net.] MatchDomains:[]}
Taildrop disabled; no state directory
peerapi starting without Taildrop directory configured
peerapi: serving on http://100.123.10.89:57209
peerapi: failed to do peerAPI listen, harmless (netstack available) but error was: listen tcp6 [fd7a:115c:a1e0:ab12:4843:cd96:627b:a59]:0: bind: cannot assign requested address
peerapi: serving on http://[fd7a:115c:a1e0:ab12:4843:cd96:627b:a59]:1
bootstrapDNS("derp3.tailscale.com", "68.183.179.66") for "controlplane.tailscale.com" = [2a05:d014:386:202:a3e1:5be6:cbb1:1ac9 2a05:d014:386:202:6dd4:fc68:35ae:2b80 2a05:d014:386:201:7200:6340:b22f:7df6 2a05:d014:386:203:4535:c15c:9ab:8258 2a05:d014:386:202:24dd:46aa:f98e:9997 2a05:d014:386:203:b7ba:b7d4:9703:6f8a 18.193.143.177 18.193.255.254 18.156.90.224 3.121.104.141 18.157.173.201 52.29.8.43]
magicsock: home is now derp-1 (nyc)
magicsock: adding connection to derp-1 for home-keep-alive
magicsock: 1 active derp conns: derp-1=cr0s,wr0s
derphttp.Client.Connect: connecting to derp-1 (nyc)
Switching ipn state Starting -> Running (WantRunning=true, nm=true)
control: NetInfo: NetInfo{varies= hairpin=false ipv6=true udp=true icmpv4=false derp=#1 portmap= link=""}
magicsock: endpoints changed: 73.114.230.110:41641 (stun), [::ffff:73.114.230.110]:41641 (stun), 10.0.3.1:41641 (local), 10.0.5.1:41641 (local), 10.0.7.1:41641 (local), 192.168.1.104:41641 (local)
magicsock: endpoints changed: [::ffff:73.114.230.110]:1384 (stun), 10.0.3.1:41641 (local), 10.0.5.1:41641 (local), 10.0.7.1:41641 (local), 192.168.1.104:41641 (local)
magicsock: derp-1 connected; connGen=1
health("overall"): ok
wgengine: Reconfig: configuring userspace WireGuard config (with 1/12 peers)
Accept: TCP{100.123.10.89:50310 > 100.97.120.100:22} 60 ok out
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b70::3eb]:41641: write udp6 [::]:41641->[2601:199:4302:3b70::3eb]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b70:211:32ff:fe8b:47f9]:41641: write udp6 [::]:41641->[2601:199:4302:3b70:211:32ff:fe8b:47f9]:41641: sendto: network is unreachable
magicsock: disco: node [JuRv3] d:434acc24204602ff now using 192.168.1.234:41641
Accept: TCP{100.97.120.100:22 > 100.123.10.89:50310} 60 tcp non-syn
Accept: TCP{100.123.10.89:50310 > 100.97.120.100:22} 52 ok out
wgengine: idle peer [BdtYK] now active, reconfiguring WireGuard
wgengine: Reconfig: configuring userspace WireGuard config (with 2/12 peers)
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:5429:9d9a:eb40:8830]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:5429:9d9a:eb40:8830]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:f9ea:b1c6:3696:ec94]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:f9ea:b1c6:3696:ec94]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71::264]:41641: write udp6 [::]:41641->[2601:199:4302:3b71::264]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:653e:dc1e:fe23:916e]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:653e:dc1e:fe23:916e]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:c41:40cf:e6d5:e80c]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:c41:40cf:e6d5:e80c]:41641: sendto: network is unreachable
magicsock: disco: node [BdtYK] d:c67ea411d06262e1 now using 192.168.10.173:41641
Accept: TCP{100.123.44.10:50148 > 100.123.10.89:445} 64 tcp ok
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50148} 60 ok out
Accept: TCP{100.123.44.10:50148 > 100.123.10.89:445} 52 tcp non-syn
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:f9ea:b1c6:3696:ec94]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:f9ea:b1c6:3696:ec94]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71::264]:41641: write udp6 [::]:41641->[2601:199:4302:3b71::264]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:c41:40cf:e6d5:e80c]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:c41:40cf:e6d5:e80c]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:5429:9d9a:eb40:8830]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:5429:9d9a:eb40:8830]:41641: sendto: network is unreachable
magicsock: disco: failed to send *disco.Ping to [2601:199:4302:3b71:653e:dc1e:fe23:916e]:41641: write udp6 [::]:41641->[2601:199:4302:3b71:653e:dc1e:fe23:916e]:41641: sendto: network is unreachable
Accept: TCP{100.123.44.10:50150 > 100.123.10.89:445} 52 tcp non-syn
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 1280 ok out
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 124 ok out
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 124 ok out
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 124 ok out
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 520 ok out
Accept: TCP{100.123.44.10:50150 > 100.123.10.89:445} 456 tcp non-syn
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 124 ok out
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 124 ok out
Accept: TCP{100.123.10.89:445 > 100.123.44.10:50150} 124 ok out
^Ctailscaled got signal interrupt; shutting down
control: client.Shutdown()
control: client.Shutdown: inSendStatus=0
control: mapRoutine: quit
control: Client.Shutdown done.
magicsock: closing connection to derp-1 (conn-close), age 2m58s
magicsock: 0 active derp conns
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:254 Protocol:0 Scope:0 Type:1 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5210 Table:254 Mark:4294967295 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:253 Protocol:0 Scope:0 Type:1 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5230 Table:253 Mark:4294967295 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:0 Protocol:0 Scope:0 Type:7 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5250 Table:0 Mark:4294967295 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
monitor: ip rule deleted: {Family:2 DstLength:0 SrcLength:0 Tos:0 Table:52 Protocol:0 Scope:0 Type:1 Flags:0 Attributes:{Dst:<nil> Src:<nil> Gateway:<nil> OutIface:0 Priority:5270 Table:52 Mark:0 Pref:<nil> Expires:<nil> Metrics:<nil> Multipath:[]}}
flushing log.
logger closing down
```

**Comment by wingcomm** [comment]
That command works and eventually the service comes up but it dies after I Ctrl-C the process in the CLI.

However, when I go to do some thing like:

```
[/share/CACHEDEV1_DATA/.qpkg/Tailscale] # ./tailscale -socket var/run/tailscale/tailscaled.sock status
failed to connect to local tailscaled; it doesn't appear to be running (sudo systemctl start tailscaled ?)
```

It appears that the service no longer works/starts on this version of QTS.

**Comment by ivokub** [comment]
So, the way Tailscale works is that the command `tailscaled` runs a background daemon which does the tunneling etc. The command `tailscale` (note without "d" in the end) is the frontend for configuring the daemon at runtime and querying status etc.

So, you would have to run the commands in parallel -- first start up `tailscaled` and if it has started without errors, run `tailscale` in second terminal. If you do it this way, are there some signs of the problems in the logs?

Usually, when everything works normally, this package starts `tailscaled` command in the background and the user has only to use `tailscale`. But if there are problems, we do not see the logs from the `tailscaled`, thats why I recommended running it from command line.

I'll check also Tailscale issues list to see some regression. In that regard -- does it work with latest QTS version + previous Tailscale version (1.28 for example?). My QNAP is other side of the continent and I get physical access to it once a year, so I really wouldn't to lose connectivity :/

**Comment by wingcomm** [comment]
Problem solved!

https://github.com/tailscale/tailscale/issues/5673#issuecomment-1250206427

**Comment by ivokub** [comment]
> Problem solved!
> 
> https://github.com/tailscale/tailscale/issues/5673#issuecomment-1250206427
> 
> @ivokub Be carful updating remotely...

Thanks! Great that you got it solved.

Yeah, if there is no way to set the flag before updating firmware, then I really cannot upgrade. Quite annoying, to be honest.

I'll keep the issue open and will add a note in the readme.

**Comment by scheini1972** [comment]
I'm sorry,
it's not the digital signature.I set the flag already. 
Unfortunately the screenshot is in German. but it's says nohting else than tailscale is not compatible with QTS 5.0.1 and please donwgrade.
![Bildschirmfoto_2022-09-18_11-04-44](https://user-images.githubusercontent.com/17107491/190894459-dc38c56c-543c-4faa-9893-f7965426609f.png)

cheers

**Comment by scheini1972** [comment]
FYI: 
Opened a bug as well on tailscale
https://github.com/tailscale/tailscale/issues/5673

**Comment by scheini1972** [comment]
As well I tried just to downgrade tailscale to 1.28., Same error.
I do not want to downgrade QTS, I'm expecting shortly 5.1. WIll update to beta and let you know the outcome.

**Comment by wingcomm** [comment]
What version of the package are you running?

**Comment by scheini1972** [comment]
> What architecture of the package are you running? x86, ARM?

I'm having a TS-453B, it's x86 64 Bit

**Comment by ivokub** [comment]
Hmm, very strange. I'll see maybe QPKG packaging has changed and can have a quick fix.

**Comment by ivokub** [comment]
ZeroTier has similar errors: https://discuss.zerotier.com/t/qnap-failed-to-install-zerotier-downgrade-qts-to-5-0-0-or-an-older-compatible-version/9346, so I guess it is wider problem with QNAP.

Just in case - do you have QVPN package installed? Can you try installing it first and then reinstall latest Tailscale?

**Comment by ivokub** [comment]
And FYI for anyone reading -- the latest Tailscale package is built with the latest version of QDK kit. CI always fetches the default QDK from source and uses it for building the QPKG package. So the issue shouldn't at least be that the package is built with old tools.

**Comment by scheini1972** [comment]
> ZeroTier has similar errors: https://discuss.zerotier.com/t/qnap-failed-to-install-zerotier-downgrade-qts-to-5-0-0-or-an-older-compatible-version/9346, so I guess it is wider problem with QNAP.
> 
> Just in case - do you have QVPN package installed? Can you try installing it first and then reinstall latest Tailscale?

I do have QVPN installed already. Can't install tailscale, same error: not compatible.

**Comment by wingcomm** [comment]
I'm running 1.30.1 and its still working on my TVS-471. Can you try that build?

**Comment by scheini1972** [comment]
Hi wingcomm,

do you have the latest firmware 5.0.**1**? Everything worked fine until QTS 5.0.**0**.2131 build 20220815
Don't upgrade, otherwise tailscale (any version) won't work.
Cheers

**Comment by wingcomm** [comment]
I'm running QTS 5.0.1.2145 on my TVS-471 and Tailscale 1.30.1 right now...how did I dodge this bullet?

**Comment by wingcomm** [comment]
I will add that I'm a tad afraid to upgrade to 1.30.2 until this issue is resolved. 😬

**Comment by scheini1972** [comment]
I uninstalled tailscale, since when not even able to install it :-( 
I just tried it again with 1.30.1, without success. :-(

Meanwhile try to workaround with a docker image.
For the future, @ŧailscale I think they need to get a digital signature anyhow. As already stated in the Release Notes. By default always switched off

**Comment by scheini1972** [comment]
I uninstalled tailscale, since when not even able to install it :-( 
I just tried it again with 1.30.1, without success. :-(

Meanwhile try to workaround with a docker image.
For the future, @ŧailscale I think they need to get a digital signature anyhow. As already stated in the Release Notes. By default always switched off

**Comment by ajnadox** [comment]
I also had the same issue and the only current solution i found was to downgrade to 5.0.0 version from Aug 2022, then i could upgrade from 1.26 to 1.30.2
not even 1.26 was running on 5.0.1 release

**Comment by scheini1972** [comment]
@ivokub : I was opening a ticket at Qnap and they were so kind to invite me to a discord channel for discussing the issue.
Perhaps you or somebody from tailscale could join ?
https://discord.com/invite/H2QaRJE72q

thanks

**Comment by ivokub** [comment]
> @ivokub : I was opening a ticket at Qnap and they were so kind to invite me to a discord channel for discussing the issue. Perhaps you or somebody from tailscale could join ? https://discord.com/invite/H2QaRJE72q
> 
> thanks

I looked around Discord. Is there a particular channel or user which was referred?

**Comment by scheini1972** [comment]
> > @ivokub : I was opening a ticket at Qnap and they were so kind to invite me to a discord channel for discussing the issue. Perhaps you or somebody from tailscale could join ? https://discord.com/invite/H2QaRJE72q
> > thanks
> 
> I looked around Discord. Is there a particular channel or user which was referred?

Hi Ivo,
I was looking around, seem no special topic, seems we would be able to start a discussion there.

**Comment by case** [comment]
Fwiw, installing the latest QVPN app, then reinstalling Tailscale 1.30.2 (and reconnecting via the commands in the Readme here) worked for me.

**Comment by scheini1972** [comment]
> Fwiw, installing the latest QVPN app, then reinstalling Tailscale 1.30.2 (and reconnecting via the commands in the Readme here) worked for me.

Hi Case. I do have QVPN 3.0.779. I tried following:
- QVPN stop, Tailscale install --> failure
- Deinstalled QVPN, Tailscale install --> failure
- Reinstalled QVPN, Tailscale install --> failure
Sorry, didn't work for me. Currently I do not have tailsacle installed at all, as it does not allow in anymore :-(

**Comment by ivokub** [comment]
> > > @ivokub : I was opening a ticket at Qnap and they were so kind to invite me to a discord channel for discussing the issue. Perhaps you or somebody from tailscale could join ? https://discord.com/invite/H2QaRJE72q
> > > thanks
> > 
> > I looked around Discord. Is there a particular channel or user which was referred?
> 
> Hi Ivo,
> I was looking around, seem no special topic, seems we would be able to start a discussion there.

I tried asking but received no reply. Will try further.

**Comment by johntait** [comment]
I initially installed Tailscale 1.26 under QTS 5.0.0 then, still under QTS 5.0.0, I updated it to Tailscale 1.30.1. Tailscale 1.30.1 is still running properly after upgrading from QTS 5.0.0 to QTS 5.0.1.2145.

Having Tailscale functional on this specific NAS isn't critical to me. If I can help with testing anything please let me know.

**Comment by JimmyTan123-dotcom** [comment]
hi all,
I have the problem for that I cannot click open !! 
even I restart NAS and also re-install 1.30.2 still cannot click "open"
---------------
condition
NAS TS-364
tailscale VPN : 1.30.2
---------------

**Comment by scheini1972** [comment]
Just for Info,
upgraded to firmware QTS 5.0.1.2173.
Still the same issue
Dieter

**Comment by johntait** [comment]
Upgraded to firmware QTS 5.0.1.2173 and Tailscale 1.30.1 continued to work properly.

Upgraded Tailscale to v1.30.2 using QNap Club package and upgrade was successful.
Rebooted to make certain Tailscale would continue to run properly.

In case it's helpful, here's the Tailscale section from qpkg.conf

[Tailscale]
Name = Tailscale
Class = null
Status = complete
store = fdb515956e895c7b287c5075b73b9b90
Build = 20220930
Display_Name = Tailscale VPN
Version = 1.30.2
Author = Qoolbox
QPKG_File = Tailscale.qpkg
Date = 2022-10-04
Shell = /share/CACHEDEV1_DATA/.qpkg/Tailscale/Tailscale.sh
Volume_Select = 3
Install_Path = /share/CACHEDEV1_DATA/.qpkg/Tailscale
RC_Number = 122
Enable = TRUE
FW_Ver_Min = 4.2.6

**Comment by filipzahradnik** [comment]
My Tailscale stopped working after upgrade to QTS 5.0.1.2145.

I just installed Tailscale 1.30.1 from QNAP Club, and Tailscale started working. It upgraded the previous installation (hand-built from this repo) cleanly and did not require re-authentication.

https://www.qnapclub.eu/en/qpkg/1162

Happy days.

**Comment by scheini1972** [comment]
By download (manually) tailscale from https://www.qnapclub.eu/en/qpkg/1162 with the newest version. 
Installing worked and tailsclae is running again on my NAS!!! :-) :-) :-)
Thank you very much.
I think the issue is with the digital signature. As the one from qnapclu.eu has it, installation went fin.

**Comment by ivokub** [comment]
That is great to know that the version from QNAPClub works. I guess I would recommend keeping using their version. It is still strange that QTS prevents installing unsigned packages when it is allowed.

**Comment by scheini1972** [comment]
Hi all,
FYI: tried once again - as above fellow @filipzahradnik  did above - having the Tailscale Package from Qnapclub installe, upgrading to 1.32.0 (from this repository). Did not work.
Reinstalled the Qnapclub-Package again.
Settings to allow unsigned packages is set, however does not work for me with Tailscale

cheers and have a nice weekend.

**Comment by wormi4ok** [comment]
I also experienced issues after upgrading to QTS 5.0.1.2145. It seems like the problem was in the `resolv.conf` for me - Tailscale overrides the content to point to internal Tailscale-managed DNS nameserver - `100.100.100.100`, which is not accessible when Tailscale is down. I've reverted the content of the file to the original state (there is a backup version right next to `/etc/resolv.conf`) and reinstalled Tailscale package manually from downloaded release archive.
So far it is working fine for me - the link survived an upgrade to QTS 5.0.1.2173 and and another manual upgrade to Tailscale 1.32.0.

**Comment by scheini1972** [comment]
Dear all, 
problem still persists after upgrade to QTS QTS 5.0.1.2194 on a TS-453B.
Clear the issue is app needs a digital signature.

cheers
Dieter

**Comment by ivokub** [comment]
Thanks for updating. I'll see if it is possible to generate a self-signed key for signing the packages. In that case, there would be a signature which wouldn't be trusted by QTS. Maybe it makes a difference compared to having no digital signature at all.

But it seems that a proper long-term solution would be to use signed package either from QNAP Club or in the future from QNAP when they include Tailscale. I think it would be too excessive to maintain a repository for providing a single package.

**Comment by wingcomm** [comment]
Just upgraded to Tailscale 1.32.2 x64 on my TVS-471 running QTS 5.0.1.2194 and it went flawlessly. I'm starting to be skeptical of OPs categorization of the issue. It appears to be more specific than just a QTS version issue.

**Comment by scheini1972** [comment]
Low priority: 
FYI: did create a support ticket with Qnap. Unfortunately I got a dissatisfying answer from them that they think it's not possible to install 3rd party apps without digital signature anymore. Whether this is only related to my TS-453B or any other is not clarified ;-).

**Comment by talios** [comment]
@scheini1972 that's weird cause I've been installing the updated releases on the current QTS all along with no issue (TS-653D), just change the setting for allowing missing sigs in the AppCenter settings:

![image](https://user-images.githubusercontent.com/19093/200515669-34642f85-fdf7-41c1-9717-9b74b2e2dadd.png)

**Comment by scheini1972** [comment]
@talios : One of the 1st things I checked :-)
![grafik](https://user-images.githubusercontent.com/17107491/200516483-1870faa9-5394-4b56-9ffb-f03387a4b015.png)

**Comment by Skip23** [comment]
Good evening,
Is it OK to upgrade to QTS 5.0.1.2194 and Tailscale still works?
Thanks in advance.

**Comment by ivokub** [comment]
> Good evening, Is it OK to upgrade to QTS 5.0.1.2194 and Tailscale still works? Thanks in advance.

There are different feedbacks. This package may or may not work. However, Tailscale from QNAPClub should work after upgrade as it has valid digital signature.

**Comment by Skip23** [comment]
> > Good evening, Is it OK to upgrade to QTS 5.0.1.2194 and Tailscale still works? Thanks in advance.
> 
> There are different feedbacks. This package may or may not work. However, Tailscale from QNAPClub should work after upgrade as it has valid digital signature.

Thanks for the quick response.
S

**Comment by scheini1972** [comment]
Dear all,

using the new package provided from https://github.com/tailscale/tailscale-qpkg/releases worked nwo to install. 
No need anymore to create the package myself.
Thanks to all

**Comment by MilkTeaNo7** [comment]
> My Tailscale stopped working after upgrade to QTS 5.0.1.2145.
> 
> I just installed Tailscale 1.30.1 from QNAP Club, and Tailscale started working. It upgraded the previous installation (hand-built from this repo) cleanly and did not require re-authentication.
> 
> https://www.qnapclub.eu/en/qpkg/1162
> 
> Happy days.

My friend, thank you for your reply. I used your method and successfully resolved the entire issue. It was strange that I encountered the "failed to connect to local tailscaled; it doesn't appear to be running" problem with version 1.41, but as you suggested, using your recommended version did indeed solve the problem. I don't want to investigate the root cause of the issue anymore. Anyway, I used your method to solve the problem. Thank you for your assistance. Warmest greetings from China.
